src/main.rs

```diff
@@ -0,0 +1,175 @@
+use typedb_client::{
+    concept::{Attribute, Concept, Value},
+    Connection, DatabaseManager, Session,
+    SessionType::{Data, Schema},
+    TransactionType::{Read, Write}
+};
+use futures::StreamExt;
+use futures::TryStreamExt;
+use  std::fs;
+use std::io;
+
+#[derive(Debug)]
+enum HandleError {
+    Io(io::Error),
+    TypeDB(typedb_client::error::Error),
+}
+
+const TEST_DATABASE: &str = "codeforces-data-model";
+ 
+fn new_core_connection() -> typedb_client::Result<Connection> {
+    Connection::new_plaintext("localhost:1729")
+}
+
+fn read_input() -> Result<String, HandleError>{
+    let mut input_string = String::new();
+    io::stdin().read_line(&mut input_string).map_err(HandleError::Io)?;
+    input_string.pop();
+    Ok(input_string)
+}
+
+async fn load_data(connection: Connection)->Result<(), HandleError>{
+    let data=fs::read_to_string("./src/data.tql").map_err(HandleError::Io)?;    
+    let databases = DatabaseManager::new(connection.clone());
+    let session = Session::new(databases.get(TEST_DATABASE).await.map_err(HandleError::TypeDB)?, Data).await.map_err(HandleError::TypeDB)?;
+    let transaction = session.transaction(Write).await.map_err(HandleError::TypeDB)?;
+    let inserted_query = transaction.query().insert(data.as_str()).map_err(HandleError::TypeDB)?;
+    let _ = inserted_query.try_collect::<Vec<_>>().await;
```

Comment:
Can we handle the error returned by `.try_collect`? If we do so, we shouldn't need to assign the return value to a variable.